src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -2174,7 +2174,8 @@ void EmitPositiveLookaroundAssertion(RegexNode node)
                 }
 
                 // Save off pos.  We'll need to reset this upon successful completion of the lookaround.
-                string startingPos = ReserveName("positivelookaround_starting_pos");
+                bool rtl = (node.Options & RegexOptions.RightToLeft) != 0;
```

Comment:
NIT: since you are only using this once, would it make more sense to make the check on the next line directly instead of saving the local?